samples/subsys/usb/webusb/sample.yaml

```diff
@@ -3,5 +3,5 @@ sample:
 tests:
   sample.usb.webusb:
     depends_on: usb_device
-    platform_whitelist: quark_se_c1000_devboard
     tags: usb
+    platform_exclude: native_posix
```

Comment:
After #17397 is merged this is not needed

Reply:
#17397 not helping, still the same issue.